src/modules/dracut/main.py

```diff
@@ -33,7 +34,12 @@ def run_dracut():
 
     :return:
     """
-    return target_env_call(['dracut', '-f'])
+    kernelName = libcalamares.job.configuration['kernelName']
+
+    if kernelName != 0:
+        return check_target_env_call(['dracut', '-f', '{}'.format(kernelName)])
```

Comment:
.. but `kernelName` is a string already, isn't it? So it's not necessary to format to string (or you could use `str()` here).

Reply:
I used it that way for consistency in the file (see line 59), but not clear to me what the standard way should be, from:
https://docs.python.org/3/library/string.html#format-string-syntax

>Changed in version 3.1: The positional argument specifiers can be omitted for [str.format()](https://docs.python.org/3/library/stdtypes.html#str.format), so '{} {}'.format(a, b) is equivalent to '{0} {1}'.format(a, b).